testapp/build.gradle

```diff
@@ -9,6 +9,13 @@ android {
         versionCode versionCode
         versionName versionName
 
+        manifestPlaceholders = [
+                baseUrl: property("MINIAPP_SERVER_BASE_URL") ?: "https://www.example.com/",
```

Comment:
Seems this implementation isn't working on CI because Gradle throws an exception when the property doesn't exist. Need to check if the property exists first.

- [ ] Check if gradle property exists before using